src/core/layertree/qgslayertreemodel.cpp

```diff
@@ -325,7 +326,8 @@ QVariant QgsLayerTreeModel::data( const QModelIndex &index, int role ) const
         const bool estimatedCount = QgsDataSourceUri( layer->dataProvider()->dataSourceUri() ).useEstimatedMetadata();
         if ( showFeatureCount && estimatedCount )
         {
-          parts << "<b>Feature Count is estimated</b> : Please consider keeping database statistics up to date";
+          parts << QStringLiteral( "<b>%1</b> %2" ).arg(
+                  tr( "Feature Count is estimated" ), tr( ": the feature count is determined by the database statistics" ) );
```

Comment:
```suggestion
                  tr( "Feature count is estimated" ), tr( ": the feature count is determined by the database statistics" ) );
```

Reply:
Are you sure the `:` exists in all language, like Japanese or Arabic ? I'm gonna make it one single transable string if ot's better this way.

Reply:
I'm not an expert on this matter. But basically, that's the reason for proposing a single translatable string (also gives more context to the translators).